### ISSUE TITLE
Is there any way to use containable behavior ? 

### ISSUE THREAD

**Issue by nitishkumar1820** [tigrang/cakephp-datatable#71]
Hi,
Is there anyway to use containable behavior in datatable query ?

**Comment by tigrang** [comment]
Yes, just add the `contain` key to the configuration in the component.

**Comment by vermarenu** [comment]
Hi,
According to above suggestion i added code like : 
'contain' => array('table name'),'joins' => array(
                        array(
                            'table' => 'table name',
                            'alias' => 'table alias',
                            'foreignKey' => false,
                            'conditions' => [],
                            'type' => 'inner',
                        ),
                    ), 

rule for this "table name" is not define in that particular model.
but nothing happen. please correct where i am wrong ? is this necessary to define rule in model ? 
Also if i want to bind/unbind model for this plug-in then where to write code.

**Comment by nitishkumar1820** [comment]
I couldn't find the way for fetch data with recursive level 2. I had tried contain as well as join queries. I also try it with setting recursive 2

**Comment by tigrang** [comment]
@nitish-mca can you share some code. Also, take a look at http://mark-story.com/posts/view/using-bindmodel-to-get-to-deep-relations

If you can get the queries that are generated that would help. Thanks.
